### ISSUE TITLE
token_type_id has only two possible value， yes？

### ISSUE THREAD

**Issue by novas-meng** [google-research/bert#984]
I read the code, found that in the comments, 
token_type_ids = tf.constant([[0, 0, 1], [0, 2, 0]])

But I think token_type_ids can only be 1 or 0 , am  I wrong

**Comment by Akshayextreme** [comment]
No. `token_type_id` can be more than 2... [token_type_vocab_size](https://github.com/google-research/bert/blob/master/modeling.py#L431) is 16 in implementation.

Check [this issue](https://github.com/google-research/bert/issues/16) for more info.
